third ludo review of group 5 by group 3 and 8

skjohansen opened the pull request.

singelmamma committed:
@singelmamma @bluemace @andremorad

singelmamma committed:
@singelmamma

Update method and gitigonre on connectionsstring json-file

singelmamma committed:
Delete appsettings.json

singelmamma committed:
@singelmamma

singelmamma committed:
Delete MyContext.cs

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma

Changed name on method

ChainsGBG commented on Source/Source/TheLudoGameApp/Classes/GameMenu.cs:
```diff
@@ -0,0 +1,105 @@
+                    }
+                    else
+                    {
+                        Console.WriteLine("You have entred wrog value");
```
Alternativt kanske ska stå "You have entered an incorrect value"

ChainsGBG commented on Source/Source/TheLudoGameApp/Classes/GameMessages.cs:
```diff
@@ -0,0 +1,79 @@
+            Console.WriteLine($"{player.PlayerName} turn");
+            Console.WriteLine($"Press any button to throw the die");
+        }
+        public static void PrintKnockOut()
+        {
+            Console.WriteLine("Not yet implemented");
+        }
```
Kanske lägga in den som day 1 dlc istället 😄

ChainsGBG reviewed (commented):
Bra att ni fått med logiken för att lämna nästet.
Lite mycket filer att gå igenom för tillfället kanske dags för lite vårstädning?
Ser bra ut ser fram emot att se det färdiga projektet.

AndreMorad committed:
@andremorad

add xUnit test projekt

singelmamma committed:
@singelmamma

AndreMorad committed:
@andremorad

AndreMorad committed:
@andremorad

AndreMorad committed:
Merge branch 'master' of https://github.com/PGBSNH19/the-ludo-game-grupp-5

singelmamma committed:
@singelmamma

added test

AndreMorad committed:
@andremorad

AndreMorad committed:
Merge branch 'master' of https://github.com/PGBSNH19/the-ludo-game-grupp-5

singelmamma committed:
@singelmamma

Deleted files

singelmamma committed:
@singelmamma

Deleted more files and added tests

singelmamma committed:
@singelmamma

Added som more unit tests and renamned some of the old tests

singelmamma committed:
@singelmamma

Rearranged GameWinner test

AndreMorad committed:
@andremorad

added KnockOutAnotherToken xUnitTest

singelmamma committed:
@singelmamma

AndreMorad committed:
Merge branch 'master' of https://github.com/PGBSNH19/the-ludo-game-grupp-5

singelmamma committed:
@singelmamma

Added intergration testfile and moq package

Bluemace committed:
@bluemace

Unittest for InGoal.

AndreMorad committed:
@andremorad

added TokensToMove unit test

AndreMorad committed:
@andremorad

added Thread.Sleep and moved it to GameMenu

AndreMorad committed:
@andremorad

Changet my test  to pass

Bluemace committed:
@bluemace

Added Tests for UpdateTurnAndRound

singelmamma committed:
@singelmamma

Added new gamemessages and changed methods call in gameflow

singelmamma committed:
@singelmamma

"Token knocked out antoher token with the same color" bugfixed

singelmamma committed:
@singelmamma

Added more unit test

AndreMorad committed:
@andremorad

some changes

singelmamma committed:
@singelmamma

singelmamma committed:
Delete appsettings.dev.json

AndreMorad committed:
@andremorad

AndreMorad committed:
Merge branch 'master' of https://github.com/PGBSNH19/the-ludo-game-grupp-5

singelmamma committed:
@singelmamma

singelmamma committed:
Merge branch 'master' of https://github.com/PGBSNH19/the-ludo-game-grupp-5

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma @andremorad  @bluemace

Uppdaterade spelflödet och testerna

singelmamma committed:
@singelmamma @andremorad

Rearranged the tests to different testfiles and added som more tests

AndreMorad committed:
@andremorad

added new test

AndreMorad committed:
@andremorad

added colors to console

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma @andremorad @bluemace

Reviderade koden och tog bort onödiga metoden, samt ny namngivning på vissa metoder och varibler

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma @andremorad  @bluemace

singelmamma committed:
@singelmamma

singelmamma committed:
@singelmamma
